03-talos.tf

```diff
@@ -106,6 +137,7 @@ data "talos_client_configuration" "this" {
   cluster_name         = var.cluster_name
   client_configuration = talos_machine_secrets.this.client_configuration
   endpoints            = module.talos_control_plane_nodes.*.public_ip
+  nodes                = module.talos_control_plane_nodes.*.private_ip
```

Comment:
Where does this come from? I don't see it in https://registry.terraform.io/providers/siderolabs/talos/0.7.0/docs/resources/machine_bootstrap. Same for `endpoints` (which was already there).